exonum/src/blockchain/mod.rs

```diff
@@ -174,8 +174,18 @@ impl Blockchain {
         Schema::new(&self.snapshot()).last_block()
     }
 
+    /// This method checks whether the genesis block has already been created.
+    pub fn is_initialized(&self) -> bool {
```

Comment:
Should this method be public? I suppose there is no harm in that, but will it be really useful? I think it is better to have minimalistic public API, because it is easy to extend it, but otherwise it will be a breaking change.